Sources/hostmgr/commands/vm/VMStart.swift

```diff
@@ -74,17 +74,18 @@ struct VMStartCommand: AsyncParsableCommand {
 
     var sharedPaths: [LaunchConfiguration.SharedPath] {
         get throws {
-            guard try FileManager.default.directoryExists(at: Paths.gitMirrorStorageDirectory) else {
-                return []
+            var paths: [LaunchConfiguration.SharedPath] = []
+
+            if try FileManager.default.directoryExists(at: Paths.gitMirrorStorageDirectory)
+                && self.withGitMirrors == true {
+                paths.append(LaunchConfiguration.SharedPath(source: Paths.gitMirrorStorageDirectory, readOnly: true))
             }
 
-            guard self.withGitMirrors == true else {
-                return []
+            if try FileManager.default.directoryExists(at: Paths.commonCredentialsDirectory) {
```

Comment:
Could be for a future improvement PR, as it's not strictly necessary to move forward with the project, but I wonder if it would make sense to provide a configuration entry for this, similar to the `withGitMirrors` config entry for the other one… 🤷

Reply:
Hmm yeah if we have this for the Git mirror, it makes sense for the common credentials. Added on 2e1260c39b46702ca45db4848ab8e5968f0da76b.

Reply:
Nice 👍 

Reminder that we'll need to update the hostmgr.json config (or whatever the config file is named) on all our hosts during deployment too to turn this to true now though 😅

Reply:
Nvm I see you already did update https://github.com/Automattic/buildkite-ci/commit/65e2ed805de7960de0d5be8a88b85f79c34c12ae 👍